pkg/kubeutils/kubeutils.go

```diff
@@ -45,6 +46,9 @@ func BackOffToDuration(backoff wait.Backoff) time.Duration {
 }
 
 func WaitForNamespace(ctx context.Context, cli client.Client, ns string) error {
+	if dryrun.Enabled() {
```

Comment:
I think we should try to avoid scattering this all around the code and having to write test code in our real functions. if we want to avoid dependency injection, would it be possible to define a kubeutils interface and then we can override a global with a mock similar to how we do kots store?

https://github.com/replicatedhq/kots/blob/main/pkg/store/store.go#L10